### PR TITLE
docs: contribution/build: Change to build with CMake

### PR DESCRIPTION
**Pull request by abetomo**

### CODE REVIEW

File: doc/source/contribution/development/build/unix_autotools.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .. -*- rst -*-
 
+:orphan:
+
```
Comment (otegami) [inline doc/source/contribution/development/build/unix_autotools.rst:4]: How about adding the link to the `/install/cmake` ?

**Review by otegami — APPROVED** [review]
Look nice to me although I added a tiny comment ! 👍🏾

**Review by otegami — APPROVED** [review]
LGTM 👍🏾

File: doc/source/contribution/development/build/unix_autotools.rst [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+.. warning::
+  GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```
Comment (kou) [inline doc/source/contribution/development/build/unix_autotools.rst:6]:
```suggestion
   Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```

File: doc/source/install/others.rst [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 :orphan:
 
+.. warning::
+  GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```
Comment (kou) [inline doc/source/install/others.rst:6]:
```suggestion
   Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```

File: doc/source/contribution/development/build/unix_autotools.rst [hunk 4]
```diff
@@ -1,5 +1,10 @@
 .. -*- rst -*-
 
+:orphan:
+
+.. warning::
+  Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```
Comment (kou) [inline doc/source/contribution/development/build/unix_autotools.rst:6]:
```suggestion
   Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```

File: doc/source/install/others.rst [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 :orphan:
 
+.. warning::
+  Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```
Comment (kou) [inline doc/source/install/others.rst:6]:
```suggestion
   Building with GNU Autotools is deprecated, you should build with CMake: :doc:`/install/cmake`
```

**Pull request merged**